releases/release-5.0.3.md

```diff
@@ -0,0 +1,127 @@
+    - 支持将内置函数 `replace()` 下推到 TiFlash [#25565](https://github.com/pingcap/tidb/pull/25565)
+    - 支持将内置函数 `unix_timestamp()`、`concat()`、`year()`、`day()`、`datediff()`、`datesub()`、`castTimeAsString()`、`concat_ws()` 下推到 TiFlash [#25564](https://github.com/pingcap/tidb/pull/25564)
+    - 修改聚合算子的代价常数 [#25241](https://github.com/pingcap/tidb/pull/25241)
+    - 支持将 `Limit` 算子下推到 TiFlash [#25159](https://github.com/pingcap/tidb/pull/25159)
+    - 支持将内置函数 `str_to_date()` 下推到 TiFlash [#25148](https://github.com/pingcap/tidb/pull/25148)
+    - 允许 MPP outer join 根据表行数选择构建表 [#25142](https://github.com/pingcap/tidb/pull/25142)
+    - 支持将内置函数 `left()`、`right()`、`abs()` 下推到 TiFlash [#25133](https://github.com/pingcap/tidb/pull/25133)
+    - 支持将 Broadcast Cartesian Join 下推到 TiFlash [#25106](https://github.com/pingcap/tidb/pull/25106)
+    - 支持将 `Union All` 算子下推到 TiFlash [#25051](https://github.com/pingcap/tidb/pull/25051)
+    - 支持 MPP 查询任务按 Region 均衡到不同 TiFlash 节点上 [#24724](https://github.com/pingcap/tidb/pull/24724)
+    - 支持执行 MPP 查询后将缓存中过时的 Region 无效化 [#24432](https://github.com/pingcap/tidb/pull/24432)
+
++ TiKV
+
+    - 限制 TiCDC sink 的内存消耗 [#10305](https://github.com/tikv/tikv/pull/10305)
+    - 为 TiCDC old value 缓存增加基于内存使用量的上限 [#10313](https://github.com/tikv/tikv/pull/10313)
+
++ PD
+
+    - 将 TiDB Dashboard 升级至 v2021.06.15.1 [#3798](https://github.com/pingcap/pd/pull/3798)
+
++ TiFlash
+
+    - 支持将 `STRING` 类型转换为 `DOUBLE` 类型
+    - 支持 `STR_TO_DATE()` 函数
+    - 通过多线程优化右外连接中的非连接数据
+    - 支持笛卡尔积 Join
+    - 支持 `LEFT()` 和 `RIGHT()` 函数
+    - 支持在 MPP 查询中自动清理过期的 Region 信息
+    - 支持 `ABS()` 函数
+
++ Tools
+
+    + TiCDC
+
+        - 优化 gRPC 的重连逻辑，提升 KV client 的吞吐 [#1922](https://github.com/pingcap/ticdc/pull/1922)
+        - 优化 sorter I/O 报错信息 [#1977](https://github.com/pingcap/ticdc/pull/1977)
+
+## Bug 修复
+
++ TiDB
+
+    - 修复在 `SET` 类型列上 Merge Join 结果不正确的问题 [#25694](https://github.com/pingcap/tidb/pull/25694)
+    - 修复 `IN` 表达式参数的数据腐蚀问题 [#25666](https://github.com/pingcap/tidb/pull/25666)
+    - 避免 GC 的 session 受全局变量的影响 [#25609](https://github.com/pingcap/tidb/pull/25609)
+    - 修复了在窗口函数查询中使用 `Limit` 时出现 panic 问题 [#25517](https://github.com/pingcap/tidb/pull/25517)
+    - 修复查询分区表时使用 `Limit` 返回错误值的问题 [#25139](https://github.com/pingcap/tidb/pull/25139)
+    - 修复了 `IFNULL` 在 `ENUM` 或 `SET` 类型上不能正确生效的问题 [#25116](https://github.com/pingcap/tidb/pull/25116)
+    - 修复了 Join 子查询中的 `count` 被改写为 `first_row` 导致结果不正确的问题 [#25062](https://github.com/pingcap/tidb/pull/25062)
+    - 修复了 `TopN` 算子下使用 `ParallelApply` 查询时卡住的问题 [#25011](https://github.com/pingcap/tidb/pull/25011)
+    - 修复了使用含有多列的前缀索引查询时出现多余结果的问题 [#24635](https://github.com/pingcap/tidb/pull/24635)
+    - 修复了操作符 `<=>` 不能正确生效的问题 [#24633](https://github.com/pingcap/tidb/pull/24633)
+    - 修复并行 `Apply` 算子的数据竞争问题 [#24345](https://github.com/pingcap/tidb/pull/24345)
+    ?- 修复在分区表上对 union IndexMerge 结果排序时出现的 `index out of range` 错误 [#24155](https://github.com/pingcap/tidb/pull/24155)
```

Comment:
@tangenta The Chinese and English versions look inconsistent (at least literally). Could you help make the two versions look closer and clearer?

The English version: "Fix the issue that the sorting on `IndexMerge` results in the partition union reports the `index out of range` error".

Reply:
```suggestion
    - 修复对 PartitionUnion 算子的 IndexMerge 结果排序时出现 `index out of range` 错误 [#24155](https://github.com/pingcap/tidb/pull/24155)
```